### PR TITLE
Laravel 9 compatibility

### LINKED ISSUE #103: Add compatibility with laravel 9.
` Problem 1
    - Root composer.json requires saintsystems/odata-client ^0.6.1 -> satisfiable by saintsystems/odata-client[0.6.1].
    - saintsystems/odata-client 0.6.1 requires illuminate/support ^5.8 || ^6.0 || ^7.0 || ^8.0 -> found illuminate/support[v5.8.0, ..., 5.8.x-dev, v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.`

### PR DESCRIPTION
**Pull request by VincentBean**
Fixes #103

### CODE REVIEW

**Comment by VincentBean** [comment]
@anderly any plans to merge this?

**Comment by shoosah** [comment]
Would be great to review/merge this PR pls :)

**Comment by ORudenkoAS** [comment]
Yes. We are waiting for this PR. Could you please review and merge it?

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         "php": "^7.3 || ^8.0",
         "guzzlehttp/guzzle": "^7.0",
         "nesbot/carbon": "^2.0",
-        "illuminate/support": "^5.8 || ^6.0 || ^7.0 || ^8.0"
+        "illuminate/support": "^5.8 || ^6.0 || ^7.0 || ^8.0 | ^9.0"
```
Comment (ORudenkoAS) [inline composer.json:19]:
```suggestion
        "illuminate/support": "^5.8 || ^6.0 || ^7.0 || ^8.0 || ^9.0"
```

Comment (ORudenkoAS) [reply composer.json:19]: Of course it will works but I think suggested change will be better.

Comment (VincentBean) [reply composer.json]: I agree, thank you!

Comment (ORudenkoAS) [reply composer.json]: Have you tried to contact with code owner about merging? Yesterday I sent email to Adam. Wait for him reaction.

Comment (VincentBean) [reply composer.json]: Yes I've done that and gotten no reply, hope you have more luck :)

**Comment by VincentBean** [comment]
We've forked this repo with this PR included for anyone that needs this: https://packagist.org/packages/justbetter/odata-client

**Comment by anderly** [comment]
@VincentBean, I don't think we need this anymore since #105 was merged, right?

**Comment by shoosah** [comment]
@anderly Does that mean this PR should be closed and then you'll create a new tagged version?

**Comment by anderly** [comment]
@shoosah 0.6.2 released.

**Comment by shoosah** [comment]
@anderly Appreciate if you could update composer as well :pray: https://composer-packages.com/packages/saintsystems/odata-client

**Comment by anderly** [comment]
@shoosah fixed that. Retagged with 0.6.2

**Pull request closed without merging**